src/Jackett.Common/Indexers/MejorTorrent.cs

```diff
@@ -425,8 +425,8 @@ private static TorznabQuery ParseQuery(TorznabQuery query)
             // query.Episode = 8
             var searchTerm = query.GetQueryString();
 
-            // replace non-english alphanumeric characters with spaces
-            // searchTerm = Doctor Who 2005
+            // Server returns a 500 error if a UTF character higher than \u00FF (ÿ) is included,
+            // so we need to strip them
```

Comment:
```suggestion
            // Server returns a 500 error if a UTF character higher than \u00FF (ÿ) is included,
            // so we need to strip them
            // searchTerm = Doctor Who 2005
```

Restore removed example.